utilslib/dr.py

```diff
@@ -296,8 +296,11 @@ def get_custom_resource_definitions(self):
     @lib.timing_wrapper
     @lib.k8s_chunk_wrapper
     @lib.retry_wrapper
-    def list_namespaces(self, limit=100, next=''):
-        return self.v1.list_namespace(limit=limit, _continue=next)
+    def list_namespaces(self, **kwargs):
```

Comment:
Can we not use args as before but add label? Its more explicit. I find that **kwargs doesn't help with what i'm supposed to pass and so then have to dig into the v1.list_namespace method

Reply:
I did it this way because the v1.list_namespace function supports a number of named arguments and I wanted to expose them all transparently but can change it back if you like?